src/app/pages/details/details.scss

```diff
@@ -47,6 +47,12 @@
         color: text-color(white);
     }
 
+    .loc-summary-text {
+        p {
+            padding-bottom: 8px;
```

Comment:
I expect this to fail lint. We use `rem`s instead of `px` for better cross-device/scaling behavior. The equivalent to `8px` is `0.8rem`.

Reply:
Bad news for me: lint doesn't complain about it, due to changing to sasslint, I presume.

Reply:
Alright, I'll change it to 0.8rem.